src/node.cc

```diff
@@ -2720,23 +2720,19 @@ static void EnvQuery(Local<String> property,
 
 static void EnvDeleter(Local<String> property,
                        const PropertyCallbackInfo<Boolean>& info) {
-  bool rc = true;
 #ifdef __POSIX__
   node::Utf8Value key(info.GetIsolate(), property);
-  rc = getenv(*key) != nullptr;
-  if (rc)
+  if (getenv(*key) != nullptr)
```

Comment:
Good catch, according to unsetenv(3)  no need for it. I'm assuming `unsetenv()` is not slower than doing the `getenv()` check first.